src/test/resources/application.conf

```diff
@@ -1,52 +1,57 @@
-# Copyright 2023 HM Revenue & Customs
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-environment: local
-
-local {
-  services {
-    host: "http://localhost"
-    service-name-frontend {
-      port: 1234
-      productionRoute: "/path"
-    }
-  }
-}
-
-dev {
-  services {
-    host: "https://www.development.tax.service.gov.uk"
-    service-name-frontend {
-      productionRoute: "/path"
-    }
-  }
-}
-
-qa {
-  services {
-    host: "https://www.qa.tax.service.gov.uk"
-    service-name-frontend {
-      productionRoute: "/path"
-    }
-  }
-}
-
-staging {
-  services {
-    host: "https://www.staging.tax.service.gov.uk"
-    service-name-frontend {
-      productionRoute: "/path"
-    }
-  }
-}
+# Copyright 2023 HM Revenue & Customs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+environment: local
+
+local {
+  services {
+    host: "http://localhost"
+
+    auth-login-stub {
+          port: 9949
+          productionRoute = "/auth-login-stub"
+        }   
+    disa-registration-frontend {
+      port: 1202
+      productionRoute: "/disa-registration-frontend"
+    }
+  }
+}
+
+dev {

```

Comment:
I don't think we deploy to development for frontend services anymore so this isn't needed